### DISCUSSION TITLE
Weird error message "JetStream initialization failed start time can not be updated"

### DISCUSSION THREAD

**Discussion by nikolai-sonin-HMAG** [nats-io/nats-server#5170] [Q&A]
Hi,

I faced with the following error while running my single .NET Core JetStream client executing with push subscription. I'd like to know what the reason for the error is and how to have it fixed. My environment is the following:
- NET Core 6
- NATS.Client 1.1
- Windows 10 22H2
- NATS JetStream server: v2.10.10

**Comment by neilalexander** [comment]
Are you trying to do a consumer update with a different `opt_start_time`?

If so, you can't update that after the consumer was created.

**Reply by nikolai-sonin-HMAG** [reply]
I don't touch 'opt_start_time' explicitly

**Reply by neilalexander** [reply]
Where are you seeing the `start time can not be updated error` exactly? In a client response or in the NATS Server logs?

**Reply by nikolai-sonin-HMAG** [reply]
I see it in the client's response but as far as I understand this message comes directly from the server

**Reply by nikolai-sonin-HMAG** [reply]
And I get it in the form of NATS.Client.JetStream.NATSJetStreamException

**Reply by neilalexander** [reply]
Can you please post a snippet that causes the error?

**Comment by derekcollison** [comment]
https://github.com/nats-io/nats-server/commit/d9ded28ba4ec623755fad34723d6c234e2e0cec2

**Reply by derekcollison** [reply]
I believe this is the issue and has been fixed.

**Reply by nikolai-sonin-HMAG** [reply]
Thank you for giving me an idea, will have a look at it!

**Reply by nikolai-sonin-HMAG** [reply]
In what version should I see the fix? [2.10.11](https://github.com/nats-io/nats-server/releases/tag/v2.10.11) ?

**Reply by derekcollison** [reply]
https://github.com/nats-io/nats-server/releases/tag/v2.10.12
